arangod/RocksDBEngine/Methods/RocksDBTrxBaseMethods.h

```diff
@@ -62,6 +62,10 @@ class RocksDBTrxBaseMethods : public RocksDBTransactionMethods {
     return (_numInserts > 0 || _numRemoves > 0 || _numUpdates > 0);
   }
 
+  uint64_t numPrimitiveOperations() const noexcept final override {
+    return _numInserts + 2 * _numUpdates + _numRemoves;
```

Comment:
`2 * _numUpdates` because an update is a remove and an insert?
A code comment would be helpful here.

Reply:
Exactly. Will add a comment.